flightsql/src/main/java/io/deephaven/server/flightsql/FlightSqlTicketResolver.java

```diff
@@ -0,0 +1,501 @@
+            case COMMAND_STATEMENT_UPDATE_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandStatementUpdate.class));
+            case COMMAND_STATEMENT_SUBSTRAIT_PLAN_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandStatementSubstraitPlan.class));
+            case COMMAND_PREPARED_STATEMENT_QUERY_TYPE_URL:
+                return execute(sessionState, unpackOrThrow(any, CommandPreparedStatementQuery.class));
+            case COMMAND_PREPARED_STATEMENT_UPDATE_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandPreparedStatementUpdate.class));
+            case COMMAND_GET_TABLE_TYPES_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandGetTableTypes.class));
+            case COMMAND_GET_CATALOGS_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandGetCatalogs.class));
+            case COMMAND_GET_DB_SCHEMAS_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandGetDbSchemas.class));
+            case COMMAND_GET_TABLES_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandGetTables.class));
+            case COMMAND_GET_SQL_INFO_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandGetSqlInfo.class));
+            case COMMAND_GET_CROSS_REFERENCE_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandGetCrossReference.class));
+            case COMMAND_GET_EXPORTED_KEYS_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandGetExportedKeys.class));
+            case COMMAND_GET_IMPORTED_KEYS_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandGetImportedKeys.class));
+            case COMMAND_GET_PRIMARY_KEYS_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandGetPrimaryKeys.class));
+            case COMMAND_GET_XDBC_TYPE_INFO_TYPE_URL:
+                return execute(unpackOrThrow(any, CommandGetXdbcTypeInfo.class));
+        }
+        throw Exceptions.statusRuntimeException(Code.UNIMPLEMENTED,
+                String.format("FlightSQL command typeUrl '%s' is unimplemented", typeUrl));
+    }
+
+    private Table execute(SessionState sessionState, CommandStatementQuery query) {
+        if (query.hasTransactionId()) {
+            throw transactionIdsNotSupported();
+        }
+        return executeSqlQuery(sessionState, query.getQuery());
+    }
+
+    private Table execute(SessionState sessionState, CommandPreparedStatementQuery query) {
+        // Hack, we are just passing the SQL through the "handle"
+        return executeSqlQuery(sessionState, query.getPreparedStatementHandle().toStringUtf8());
+    }
+
+    private Table executeSqlQuery(SessionState sessionState, String sql) {
+        // See SQLTODO(catalog-reader-implementation)
+        // final QueryScope queryScope = sessionState.getExecutionContext().getQueryScope();
+        // Note: ScopeTicketResolver uses from ExecutionContext.getContext() instead of session...
```

Comment:
> // Note: ScopeTicketResolver uses from ExecutionContext.getContext() instead of session...
Proper access to QL and QS (etc) is through proper registration of the ExecutionContext. I see that we are passing the sessionstate here, but potentially we could add features where there is more than one execution context accessible to a session.

I think we probably should keep it through ExecCtx.